NOTICE

```diff
@@ -1,5 +1,6 @@
-Copyright 2020 My.com B.V. (Mail.Ru Group)
-Copyright 2021-2025 Organic Maps Contributors
+Copyright 2010-2021 My.com B.V. (Mail.Ru Group)
```

Comment:
Why 2010? There were different copyright holders.
```suggestion
Copyright 2014-2021 My.com B.V. (Mail.Ru Group)
```

Reply:
My impression was that all the codebase copyrights were transferred to Mail.ru as a part of the acquisition deal. So it had been the legal owner of MM copyrights since then. No?

Reply:
The transfer happened in 2014, "since then" means 2014 and later.

Reply:
To the best of my knowledge the years here don't represent history of copyright ownership change, i.e. `2010-2021 (c) copyright holder` means that copyrights for all changes made in 2010-2021 belong to the copyright holder now.

Reply:
> Convexity Holdings AG

you're right those copyrights don't belong to Mail.ru anymore!

Reply:
though I thought that a new holder is Daegu Limited or something

Reply:
They don't have copyrights as they've assigned them to their employer.
But its fine still to list them as authors IMHO.
Maybe it'd make sense to add a note they were payed employees and hold no copyright...

Reply:
Unfortunately, we don't have anyone from Mail.Ru Group here to provide a definitive answer. The source code was published in 2015 by Mail.Ru Group, and I believe referencing the period 2015–2021 is perfectly reasonable. Thanks to Mail.Ru Group for making it open.